### PR TITLE
fix(kalendar-git): fix build with pacstall 3 and Ubuntu 22.10

### PR DESCRIPTION
**Pull request by Zahrun**
Compiles fine in Kubuntu 22.10 (will not work in versions < 22.10 nor debian)

### CODE REVIEW

File: packages/kalendar-git/kalendar-git.pacscript [hunk 1]
```diff
@@ -22,6 +22,10 @@ build() {
 }
 
 install() {
+  cd build || exit
   sudo cmake --install .
   sudo install -Dm644 "../LICENSES/"* -t "${STOWDIR}/${name}/usr/share/licenses/${name}/"
+  cd "${STOWDIR}/${name}/usr/lib/x86_64-linux-gnu/" || exit
```
Comment (Henryws) [inline packages/kalendar-git/kalendar-git.pacscript:28]:
```suggestion
  cd "${STOWDIR}/${name}/usr/lib/x86_64-linux-gnu/"
```

File: packages/kalendar-git/kalendar-git.pacscript [hunk 2]
```diff
@@ -22,6 +22,10 @@ build() {
 }
 
 install() {
+  cd build || exit
```
Comment (Henryws) [inline packages/kalendar-git/kalendar-git.pacscript:25]:
```suggestion
  cd build
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
CI fails

**Comment by Zahrun** [comment]
> CI fails

CI does not run in Ubuntu 22.10
```
See also "/tmp/pacstall/kalendar/build/CMakeFiles/CMakeOutput.log".
  with requested version "5.96.0".

  The following configuration files were considered but not accepted:

    /usr/share/ECM/cmake/ECMConfig.cmake, version: 5.92.0
```
However, https://packages.ubuntu.com/kinetic/extra-cmake-modules

**Review by Henryws — APPROVED** [review]

**Pull request merged**